src/fragments/Chat.tsx

```diff
@@ -42,13 +70,46 @@ const Chat: React.FC = () => {
     >
       <ChatHeader channel={channel} />
       <ChatMessages channel={channel} />
+      {typingUser && (
```

Comment:
You should condition this render on `typingUsers` and remove `typingUser` , since it isn't used in the rendering logic.